archive/frames/models.py

```diff
@@ -34,88 +36,159 @@ class Frame(models.Model):
         null=True,
-        help_text="Time of observation in UTC. FITS header: DATE-OBS",
-        verbose_name="DATE-OBS"
+        help_text="Time of observation in UTC",
+    )
+    observation_day = models.DateField(
+        null=True,
+        help_text="Observing Night in YYYYMMDD",
     )
     DAY_OBS = models.DateField(
         null=True,
-        help_text="Observing Night in YYYYMMDD. FITS header: DAY-OBS",
-        verbose_name="DAY-OBS"
+        help_text="Observing Night in YYYYMMDD",
+    )
+    proposal_id = models.CharField(
+        max_length=200,
+        default='',
+        blank=True,
+        help_text="Textual proposal id"
     )
     PROPID = models.CharField(
         max_length=200,
         default='',
         blank=True,
-        help_text="Textual proposal id. FITS header: PROPID"
+        help_text="Textual proposal id"
+    )
+    instrument_id = models.CharField(
+        max_length=64,
+        default='',
+        help_text="Instrument used"
     )
     INSTRUME = models.CharField(
         max_length=64,
         default='',
-        help_text="Instrument used. FITS header: INSTRUME"
+        help_text="Instrument used"
+    )
+    target_name = models.CharField(
+        max_length=200,
+        db_index=True,
+        default='',
+        blank=True,
+        help_text="Target object name"
     )
     OBJECT = models.CharField(
         max_length=200,
         db_index=True,
         default='',
         blank=True,
-        help_text="Target object name. FITS header: OBJECT"
+        help_text="Target object name"
+    )
+    reduction_level = models.SmallIntegerField(
+        default=0,
+        help_text="Reduction level of the frame"
     )
     RLEVEL = models.SmallIntegerField(
         default=0,
         help_text="Reduction level of the frame"
     )
+    site_id = models.CharField(
+        default='',
+        max_length=3,
+        help_text="Originating site code. Usually the 3 character airport code of the nearest airport"
+    )
     SITEID = models.CharField(
         default='',
         max_length=3,
-        help_text="Originating site. FITS header: SITEID"
+        help_text="Originating site code. Usually the 3 character airport code of the nearest airport"
+    )
+    telescope_id = models.CharField(
+        default='',
+        max_length=4,
+        help_text="Originating telescope 4 character code. Ex. 1m0a or 0m4b"
     )
     TELID = models.CharField(
         default='',
         max_length=4,
-        help_text="Originating telescope. FITS header: TELID"
+        help_text="Originating telescope 4 character code. Ex. 1m0a or 0m4b"
+    )
+    exposure_time = models.FloatField(
+        null=True,
+        help_text="Exposure time, in seconds"
     )
     EXPTIME = models.DecimalField(
         null=True,
         max_digits=13,
         decimal_places=6,
         help_text="Exposure time, in seconds. FITS header: EXPTIME"
     )
+    primary_filter = models.CharField(
```

Comment:
This might be better (albeit longer) as `primary_optical_element` or something like that since it's not just filters that can go in here, it's slits and whatnot as well